Sofa/Component/Constraint/Lagrangian/Correction/src/sofa/component/constraint/lagrangian/correction/PrecomputedConstraintCorrection.inl

```diff
@@ -873,7 +873,6 @@ void PrecomputedConstraintCorrection< DataTypes >::rotateConstraints(bool back)
 
     for (auto rowIt = c.begin(); rowIt != rowItEnd; ++rowIt)
     {
-        auto rowWrite = c.writeLine(rowIt.index());
```

Comment:
Beware here... the warning indicates that rowWrite is not used... not that writeLine is useless. Unless there is proves that writeLine cannot do anything then we need to keep it.

Reply:
I agree ; nevertheless I think we can remove the warning with [[maybe_unused]]